sway/config/output.c

```diff
@@ -387,13 +387,57 @@ static void set_hdr(struct wlr_output *output, struct wlr_output_state *pending,
 	wlr_output_state_set_image_description(pending, &image_desc);
 }
 
+static struct wlr_color_transform *color_profile_from_device(struct wlr_output *wlr_output,
+	struct wlr_color_transform *transfer_function)
+{
```

Comment:
Style nit: `{` should be on the same line as the previous one (and continuation lines for parentheses should use two tabs to be able to tell the difference with the block)